app/page-partials/my-dashboard/RoleManagement/RoleEnvironment.tsx

```diff
@@ -311,129 +313,122 @@ const RoleEnvironment = ({ environment, integration, alert }: Props) => {
     setRightPanelTab(key);
   };
 
+  const activateRow = (request: any) => {
+    setSelctedRole(request['cells'][0].value);
+  };
+
   let rightPanel = null;
   if (firstUser === 0 && userLoading) {
     rightPanel = <LoaderContainer />;
   } else if (selectedRole) {
     if (rightPanelTab === 'Users') {
       rightPanel = (
-        <Table variant="mini">
-          <thead>
-            <tr>
-              <th>IDP</th>
-              <th>GUID</th>
-              <th>Email</th>
-              <th className="text-center">Actions</th>
-            </tr>
-          </thead>
-          {users.filter((user) => !user.username.startsWith('service-account-')).length > 0 ? (
```

Comment:
Is this filter working on the the new table

Reply:
added on